app/components/subcase/subcase-titles-panel.js

```diff
@@ -8,10 +8,12 @@ import { action } from '@ember/object';
  */
 export default class SubcaseTitlesPanel extends Component {
   @tracked isEditing = false;
+  @tracked startEditingConfidential = false;
 
   @action
   startEditing() {
     this.isEditing = true;
+    this.startEditingConfidential = !!this.args.subcase.confidential;
```

Comment:
I get the idea but we just need to check the toggling in the edit component. Will change this a bit